test/test_transforms_v2_refactored.py

```diff
@@ -860,3 +862,437 @@ def test_transform_noop(self, input_type, device):
         output = transform(input)
 
         assert_equal(output, input)
+
+
+class TestAffine:
+    def _make_input(
+        self, input_type, *, dtype=None, device="cpu", spatial_size=(17, 11), mask_type="segmentation", **kwargs
+    ):
+        if input_type in {torch.Tensor, PIL.Image.Image, datapoints.Image}:
+            input = make_image(size=spatial_size, dtype=dtype or torch.uint8, device=device, **kwargs)
+            if input_type is torch.Tensor:
+                input = input.as_subclass(torch.Tensor)
+            elif input_type is PIL.Image.Image:
+                input = F.to_image_pil(input)
+        elif input_type is datapoints.BoundingBox:
+            kwargs.setdefault("format", datapoints.BoundingBoxFormat.XYXY)
+            input = make_bounding_box(
+                dtype=dtype or torch.float32,
+                device=device,
+                spatial_size=spatial_size,
+                **kwargs,
+            )
+        elif input_type is datapoints.Mask:
+            if mask_type == "segmentation":
+                make_mask = make_segmentation_mask
+                default_dtype = torch.uint8
+            elif mask_type == "detection":
+                make_mask = make_detection_mask
+                default_dtype = torch.bool
+            input = make_mask(size=spatial_size, dtype=dtype or default_dtype, device=device, **kwargs)
+        elif input_type is datapoints.Video:
+            input = make_video(size=spatial_size, dtype=dtype or torch.uint8, device=device, **kwargs)
+
+        return input
+
+    def _adapt_fill(self, value, *, dtype):
```

Comment:
This will likely need to be factored out later, since we need this for every transform that supports `fill`.